specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/1.0/appconfiguration.json

```diff
@@ -1333,7 +1211,44 @@
       "type": "string",
       "x-ms-skip-url-encoding": true,
       "x-ms-parameter-location": "client"
-    }
+    },
+    "SelectSettings": {
+      "name": "$Select",
+      "in": "query",
+      "description": "Used to select what fields are present in the returned resource(s).",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "key",
+          "label",
+          "content_type",
+          "value",
+          "last_modified",
+          "tags",
+          "locked",
+          "etag"
+        ],
+        "x-ms-client-name": "SettingFields"
+      },
+      "x-ms-parameter-location": "method",
+      "collectionFormat": "csv"
+    },
+    "SelectName": {
+        "name": "$Select",
+        "in": "query",
+        "description": "Used to select what fields are present in the returned resource(s).",
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "name"
+          ],
+          "x-ms-client-name": "LabelField"
```

Comment:
I know there's only one value at present, but perhaps the name should be `LabelFields` to allow for future expansion? Might be good to follow up with the service team about the possibility.